scripts/format_new.py

```
@@ -0,0 +1,170 @@
+            phone=contact_data.get("phone"),
+            emails=contact_data.get("emails", [])
+        )
+    
+    def save_contact(self, contact: Contact) -> None:
+        """Save contact to store."""
+        self.contacts[contact.name] = {
+            "phone": contact.phone,
+            "emails": contact.emails
+        }
+        
+        with open(self.file_path, "w") as f:
```

Ruff (0.14.2)

`open()` should be replaced by `Path.open()`

(PTH123)